### ISSUE TITLE
TensorRT conversion via docker image 2.6.0-gpu - getInferLibVersion symbol not found

### ISSUE THREAD

**Issue by yotamtevel** [tensorflow/tensorflow#52196]
Trying to convert a tf.keras model to tftrt.

- CUDA/cuDNN version: 11.2
- GPU model and memory: GeForce RTX 2070

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

v2.6.0-rc2-32-g919f693420e 2.6.0

**Describe the current behavior**
` from tensorflow.python.compiler.tensorrt import trt_convert as trt
    precision = 'FP16'
    params = tf.experimental.tensorrt.ConversionParams(precision_mode=precision, use_calibration=False, minimum_segment_size=3)
    converter = trt.TrtGraphConverterV2(input_saved_model_dir=model_path, conversion_params=params)
    calibration_input_fn_arg = None
    converter.convert(calibration_input_fn_arg)
    converter.save(trt_save_path)`
**Describe the expected behavior**
2021-09-30 09:41:30.493064: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvrtc.so.11.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64
2021-09-30 09:41:30.493081: F tensorflow/compiler/tf2tensorrt/stub/nvinfer_stub.cc:49] getInferLibVersion symbol not found.
Aborted (core dumped)
**[Contributing](https://www.tensorflow.org/community/contribute)**

**Comment by mohantym** [comment]
Hi @yotamtevel! Can you please share a link to model file?

**Comment by yotamtevel** [comment]
Hi @mohantym
Those doe's not work.

Same code and same model using nvcr.io/nvidia/tensorflow:21.07-tf2-py3 works just fine - note that it downgrades tensorflow to 2.5.0+nv

**Comment by mohantym** [comment]
Ok @yotamtevel! Could you please share the link to **model_path** too to replicate the issue?

**Comment by yotamtevel** [comment]
Hi @mohantym,

`
import os
import tensorflow as tf


def mish(input_tensor):
    return input_tensor * tf.keras.activations.tanh(tf.math.log(1 + tf.math.exp(input_tensor)))


def build_model():

    input_tensor = tf.keras.layers.Input((None, None, 3), name='input')
    scaled_input = tf.math.multiply(input_tensor, 1. / 255.)
    d1 = tf.keras.layers.Conv2D(32, kernel_size=3, padding='SAME', strides=2)(scaled_input)
    d1 = mish(d1)
    d2 = tf.keras.layers.Conv2D(64, kernel_size=3, padding='SAME', strides=2)(d1)
    d2 = mish(d2)

    u1 = tf.keras.layers.UpSampling2D()(d2)
    u1 = tf.keras.layers.Conv2D(32, kernel_size=3, padding='SAME')(u1)
    u1 = mish(u1)
    u1 = tf.keras.layers.Concatenate()([u1, d1])

    output = tf.keras.layers.Conv2D(2, kernel_size=3, padding='SAME', activation='softmax')(u1)

    model = tf.keras.models.Model(input_tensor, output)
    return model


if __name__ == '__main__':
    model_save_load_path = ''  # TODO change
    precision = 'FP16'
    model = build_model()
    base_model_save_path = os.path.join(model_save_load_path, 'tfkeras')
    model.save(base_model_save_path)

    from tensorflow.python.compiler.tensorrt import trt_convert as trt

    params = tf.experimental.tensorrt.ConversionParams(precision_mode=precision.upper(),
                                                       use_calibration=False, minimum_segment_size=3)
    converter = trt.TrtGraphConverterV2(input_saved_model_dir=base_model_save_path,
                                        conversion_params=params)

    calibration_input_fn_arg = None

    converter.convert(calibration_input_fn_arg)
    converter.save(os.path.join(model_save_load_path, precision))
    print('done')

`

**Comment by mohantym** [comment]
Hi @Saduf2019 ! Could you please look at this issue ?User has this issue in Docker. The above code is running without error in Colab.Attaching [GIST](https://colab.sandbox.google.com/drive/1H2dw78EZHBMT4O6tHSpgUzfGR6oS7vfv?resourcekey=0-8zAYiBhUN9Zz_DgKfQXASw#scrollTo=l0jV8xA2Vibv) for reference along this [thread ](https://forums.developer.nvidia.com/t/tf-trt-optimization/179447/5 )with similar Error Stack Trace
